recipes/ftxui/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import conan.tools.files
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class FTXUIConan(ConanFile):
+    name = "ftxui"
+    description = "💻 C++ Functional Terminal User Interface."
+    license = "MIT"
+    topics = ("ncurses", "terminal", "screen", "tui")
+    homepage = "https://github.com/ArthurSonzogni/FTXUI"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _validate_compiler_settings(self):
+        compiler = self.settings.compiler
+        version = tools.Version(self.settings.compiler.version)
+        if compiler == 'gcc' and version < '8':
+            raise ConanInvalidConfiguration("gcc 8 required")
+        if compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "17")
+
+    def validate(self):
+        self._validate_compiler_settings()
+        if self.settings.compiler in ["Visual Studio", "msvc"]:
+            raise ConanInvalidConfiguration("Visual Studio unsupported")
```

Comment:
https://github.com/ArthurSonzogni/FTXUI/blob/5da7b8a59ae812002c52bc0459947e90ec744eb4/cmake/ftxui_set_options.cmake#L18 is visual studio supported ?

Reply:
It's experimental per the README, and since I don't have a way to test it, I'd rather not support it for now.